scripts/docs.sh

```diff
@@ -23,10 +23,19 @@ function buildDocs {
     npm run docs
 }
 
+function createGhPages {
+    git checkout --orphan $TARGET_BRANCH
+    git rm --cached -r .
+    git clean -df
+    touch README.md
+    git add README.md
+    git commit -m "Initial Commit"
+    git push $SSH_REPO $TARGET_BRANCH
+}
+
 # Pull requests and commits to other branches shouldn't try to deploy, just build to verify
 if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
```

Comment:
We can remove this check, now that the travis.yml has a condition which only runs the entire script on tags